backends/plugins/sdl/sdl-provider.cpp

```diff
@@ -22,7 +22,7 @@
 #include "common/scummsys.h"
 
 // RiscOS uses its own plugin provider and SDL one doesn't work
-#if defined(DYNAMIC_MODULES) && defined(SDL_BACKEND) && !defined(RISCOS)
+#if defined(DYNAMIC_MODULES) && defined(SDL_BACKEND)
```

Comment:
Since this was introduced in SDL 1.2.4, you might want to check for that instead of KolibriOS specifically.

Reply:
I just avoid linking it in. And even if they update SDL we still want to use native code as they have 2 different loaders and we want specifically newlib one.